### PR TITLE
[do not merge] Update Deny All to Allow All

### PR DESCRIPTION
**Pull request by dimassevfc**

### CODE REVIEW

**Comment by adellape** [comment]
@dimassevfc Which install method and version (Origin or Enterprise) were you using when you observed this?

It looks like Ansible still sets it to `deny_all` for Enterprise, otherwise `allow_all`:

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_facts/library/openshift_facts.py#L349-L358

@sdodson Do you expect this to continue to be the case going forward in OSE?

**Comment by sdodson** [comment]
Yeah, that was a feature request to make enterprise denyall by default.
Users may configure their desired auth providers by setting
openshift_master_identity_providers

File: admin_guide/configuring_authentication.adoc [hunk 1]
```diff
@@ -26,8 +26,8 @@ ifdef::openshift-enterprise[]
 link:../install_config/install/quick_install.html[Quick Installation] or
 endif::[]
 link:../install_config/install/advanced_install.html[Advanced Installation] method, the
-link:#DenyAllPasswordIdentityProvider[Deny All] identity provider is used by
-default, which denies access for all user names and passwords. To allow access,
+link:#AllowAllPasswordIdentityProvider[Allow All] identity provider is used by
+default, which allow access for all user names and passwords. To change access,
```
Comment (adellape) [inline admin_guide/configuring_authentication.adoc]: s/allow/allows/

**Comment by adellape** [comment]
@dimassevfc Per the above comments, this will need to be conditionalized for OpenShift Origin vs OpenShift Enterprise. See `ifdef/endif` statements elsewhere in this topic, as well as https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/contributing.adoc#conditional-text-between-products.

**Pull request closed without merging**